common/scripts/parse-py-metadata.py

```diff
@@ -114,6 +114,16 @@ def parse_depends(args):
     vpkgs = dict()
     extras = set(args.extras.split())
 
+    with args.provides.open() as f:
+        for ln in f.readlines():
+            if not ln.strip():
+                continue
+            pkgver, vpkgver = ln.split(" ", maxsplit=2)
```

Comment:
alternatively (but not really different)
```suggestion
            pkgver, _, vpkgver = ln.partition(" ")
```